pagopa/io-app
# feat: [IOBP-48] Discount initiative details update

## Short description
added a type distinction in an initiative's details screen
<img width="385" alt="image" src="https://github.com/pagopa/io-app/assets/60693085/4153e7e0-3eb8-441d-89ce-b06ed8ce0996">


## List of changes proposed in this pull request
- added type distinction with its required table entries
- added tests to ensure correct rendering

## How to test
yarn jest, also with `io-dev-server` navigate to an initiative's details (initiative screen>press the ( i ) logo in the top right) and make sure the rendered entries match design, both for discount and refund type initiatives
